system-test/compute.js

```diff
@@ -211,6 +211,46 @@ describe('Compute', () => {
       });
       assert.strictEqual(fetched.shieldedInstanceConfig.enableSecureBoot, true);
     });
+
+    it('API errors propagated', async () => {
```

Comment:
nit: I like for the assertions to read like a phrase.

`it('propagates API errors')`.